official/mnist/mnist.py

```diff
@@ -261,6 +264,4 @@ def __init__(self):
 
 if __name__ == '__main__':
   tf.logging.set_verbosity(tf.logging.INFO)
-  parser = MNISTArgParser()
-  FLAGS, unparsed = parser.parse_known_args()
-  tf.app.run(main=main, argv=[sys.argv[0]] + unparsed)
```

Comment:
I am bit hesitate to get rid of tf.app.run(). Usually app.run contains most of the init code that is common to all the models, and it makes the lib owner easily to add a common dependencies without modifying all the client code.

Reply:
Discussed offline.